src/data/faq.json

```diff
@@ -34,7 +34,19 @@
             {
                 "title": "Common Issues",
                 "id": "most_asked",
-                "accordion": [
+                "accordion": [
+                    {
+                        "title": "Cause: 2001, trying to e establish a secure connection to the server",

```

Comment:
trying to e establish
=> trying to establish

There is an extra "e" in the title.

Reply:
ok